Rakefile

```diff
@@ -32,6 +32,21 @@ task :vendor_deps do
   sh 'bower install'
   sh 'cp', 'bower_components/test_track_js_client/dist/testTrack.bundle.min.js', 'app/assets/javascripts'
 
+  # Download testtrack-cli
+  TEST_TRACK_CLI_VERSION = 'v0.9.7'
```

Comment:
So if we want to upgrade the vendored CLI version, do we just change this constant?

Reply:
yep, and then run `bundle exec rake vendor_deps`... there's a bit of noise w/ fakeable_her.  we are not running the latest for reasons that i don't understand (you actually addressed it in #65). so you need to not commit those changes.

Reply:
should we rip out the rest of the gem upgrades and let them rot?

Reply:
works for me, we haven't touched those gems in a long time. i can do this separately.